packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -26,44 +26,41 @@
                     :class="$style['c-cookieBanner-title']"
                     tabindex="0"
                     data-consent-title>
-                    {{ $t('mainTitle') }}
+                    {{ copy.mainTitle }}
                 </h2>
 
                 <p :class="$style['c-cookieBanner-text']">
-                    {{ $t('textLine1') }}
+                    {{ copy.textLine1 }}
                 </p>
 
                 <p :class="$style['c-cookieBanner-text']">
-                    {{ $t('textLine2') }}
+                    {{ copy.textLine2 }}
                 </p>
 
-                <p
-                    :class="$style['c-cookieBanner-text']">
-                    <i18n
-                        path="textLine3">
-                        <template #cookiePolicy>
-                            <a
-                                :href="$t('cookiePolicyLinkUrl')"
-                                target="_blank">
-                                <span>{{ $t('cookiePolicyLinkText') }}</span>
-                            </a>
-                        </template>
-                    </i18n>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.textLine3 }}
```

Comment:
I'd be tempted to use a [custom filter](https://v1.vuejs.org/guide/custom-filter.html) for this component only to scaffold the link into `textLine3` in its existing form and avoid having to split it into `textLine3` and `textLine4` in all the language files, only to merge them back together again when `f-globalisation` has been fixed. Happy to accept like this though.

Reply:
@benwhite-deltas It was done like that, but in pulling out the f-globalisation mixin, I opted to revert to the way I'd done it before